.github/workflows/nodejs.yml

```diff
@@ -0,0 +1,94 @@
+# Copyright 2010 New Relic, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+---
+
+name: "NodeJs Init Container CI"
+
+on:
+  pull_request:
+    paths:
+      - 'src/nodejs/**'
+      - '.github/workflows/nodejs.yml'
+  push:
+    paths:
+      - 'nodejs/**'
+      - '.github/workflows/nodejs.yml'
+    branches:
+      - main
+    # Do not run when a tag is created.
+    tags-ignore:
+      - "**"
+  release:
+    types:
+      - published
+
+env:
+  INITCONTAINER_LANGUAGE: nodejs
+  K8S_OPERATOR_IMAGE_TAG: edge
+
+
+jobs:
+  call-test:
+    uses: ./.github/workflows/test.yml
+    with:
```

Comment:
You'll need to add `secrets: inherit` here in order for `test.yml` to have access to Github secrets.

Reply:
And also on the call to `publish.yml`